fastavro/_schema.pyx

```diff
@@ -28,8 +28,7 @@ cpdef inline str extract_logical_type(schema):
     rt = schema['type']
     lt = schema.get('logicalType')
     if lt:
-        # TODO: Building this string every time is going to be relatively slow.
-        return '{}-{}'.format(rt, lt)
+        return rt + '-' + lt
```

Comment:
Is this change significant with respect to performance? If so it may require more work as there is currently some tests failing due to it: https://travis-ci.org/fastavro/fastavro/jobs/605518459?utm_medium=notification&utm_source=github_status

Reply:
It has only a very slight effect. I just used it because I saw it as an easy win. Reverting
```
In [45]: rt = 'bytes'                                                                                                                                                                                                                         

In [46]: lt = 'decimal'                                                                                                                                                                                                                       

In [47]: timeit rt + '-' + lt                                                                                                                                                                                                                 
79.5 ns ± 1.42 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [48]: timeit '{}-{}'.format(rt, lt)                                                                                                                                                                                                        
172 ns ± 3.59 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```